utilities/conv_gefs2datm/0Read_me_missing_GEFS

```diff
@@ -0,0 +1,233 @@
+#---- Followings are missing or failed in opening GEFS NEMSio data from HPSS archive
+#---- hyun-chul.lee@noaa.gov
+
+Followings were all recovered from errors,  by gary.bates@noaa.gov
+
+
+2000103100 --> recovered (01/22)
+2000103118 --> recovered (01/23)
+2001121118 --> recovered (01/29)
+2004062712 --> recovered (01/29)
+2004062718 --> recovered (01/29)
+2004100500 --> recovered (01/29)
+2004100518 --> recovered (01/29)
+2005080512 --> recovered (01/29)
+2005082000 --> recovered (01/29)
+2005082200 --> recovered (01/29)
+2009080612 --> recovered (02/04)
+2009080618 --> recovered (02/04)
+2009080700 --> recovered (02/04)
+2010121512 --> recovered (02/04)
+2012120212 --> recovered (01/23)
+2013051800 --> recovered (02/04)
+2015022712 --> recovered (02/04)
+2015083018 --> recovered (02/04)
+2016043000 --> recovered (02/04)
+2017031512 --> recovered (02/04)
+2017031618 --> recovered (02/04)
+2017031812 --> recovered (02/04)
+2018041700 --> recovered (01/22)
+2019070806 --> recovered (02/04)
+
+
+#---- massages in error
```

Comment:
Do we have all of the GEFS files now?

Reply:
Yes, all missing files in GEFS were recovered by the rerun.

Reply:
Then do we still need this file?

Reply:
These files have been processed completely, and saved to HPSS, Hera and DATM_INPUT.

Reply:
That's great! I don't think we need this file in the repository then, correct?

Reply:
Correct, we dont't need these files in the repository

Reply:
Did I miss where you deleted this file then?